projects/Mallard/src/helpers/files.ts

```diff
@@ -50,17 +51,19 @@ const fileName = (path: string) => {
 export const getJson = (path: string) =>
     RNFetchBlob.fs.readFile(path, 'utf8').then(d => JSON.parse(d))
 
-export const downloadNamedIssueArchive = (
+export const downloadNamedIssueArchive = async (
     localIssueId: Issue['localId'],
     assetPath: string,
     filename: string,
 ) => {
-    const zipUrl = defaultSettings.zipUrl
+    const apiUrl = await getSetting('apiUrl')
+    const zipUrl = `${apiUrl}${assetPath}`
+    console.log(`Downloading from ${zipUrl}`)
```

Comment:
Will remove this in a second.

Reply:
will apiUrl have / at the end ?

Reply:
Yep - this `console.log` showed me I had too much `/` which is why there isn't any `/` any more.